test/async-hooks/init-hooks.js

```diff
@@ -212,13 +212,13 @@ class ActivityCollector {
 }
 
 exports = module.exports = function initHooks({
-    oninit,
-    onbefore,
-    onafter,
-    ondestroy,
-    allowNoInit,
-    logid,
-    logtype } = {}) {
+  oninit,
```

Comment:
Uhh I'd prefer the form in the PR, because imagine what that variant would look like if I were to add another option.